distribution/helm/vector-aggregator/values.yaml

```
@@ -191,6 +193,8 @@ vectorSource:
     # option: "value"
   # Raw TOML config to embed at the vector source (deprecated).
   rawConfig: null
+  # Specific node port to bind if `service.type` is set to `NodePort`.
+  nodePort: 0
```

```suggestion
  nodePort: null  # 9090 (must be a number)
```

Not sure how it would play together with the changes at https://github.com/timberio/vector/pull/6020/files#r562479964, so it might be useful to change like this.